src/main/java/deanery/RoomData.java

```diff
@@ -0,0 +1,22 @@
+package deanery;
+
+import deanery.room.FullTimeRoom;
+import deanery.room.PartTimeRoom;
+import deanery.room.Room;
+import deanery.student.FullTimeStudent;
+import deanery.student.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RoomData {
+    public static List<Room> createRooms(){
+        Room fullTimeRoom1 = new FullTimeRoom("A123", 8, 16);

```

Comment:
As we know for sure what kind of rooms exist, the Room class could be an enum -> then you wouldn't need to write a specific code that would initialize it and just use the enum values